exchanges/okx/okx_test.go

```diff
@@ -2946,7 +2824,7 @@ func TestWsPlaceOrder(t *testing.T) {
 	t.Parallel()
 	sharedtestvalues.SkipTestIfCredentialsUnset(t, ok, canManipulateRealOrders)
 
-	var resp WsPlaceOrderInput
+	var resp WsOperationInput
```

Comment:
I am getting a lot of these for websocket tests
` error: Okx websocket connection: cannot send message to a disconnected websocket`
they are not occurring on `master` branch